dom/test/browser/events.ts

```diff
@@ -885,15 +886,15 @@ describe('DOMSource.events()', function() {
       DOM: makeDOMDriver(createRenderTarget()),
     });
     const event$ = sources.DOM.select('.myelementclass').events('click');
-    assert.strictEqual(event$._isCycleSource, 'DOM');
+    assert.strictEqual((event$ as any)._isCycleSource, 'DOM');
     done();
   });
 
   it('should allow restarting of event streams from isolated components', function(done) {
     const outSubject = xs.create<any>();
     const switchSubject = xs.create<any>();
 
-    function component(_sources: {DOM: DOMSource}) {
+    function component(_sources: {DOM: MainDOMSource}) {
```

Comment:
Why is this needed? A change in tests usually indicates a breaking change and migration notes

Reply:
This was by mistake